src/core/test/test_query_rewriter.py

```diff
@@ -68,7 +67,7 @@ def test_extract_table_token(self):
                  "where ... ")
         token = sqlparse.parse(query)[0].tokens[6]
         expected_result = [(["repo1", "table1", None], "as tbl1"),
-                           (["repo2", "table2", None], "as tbl2")]
+                           (["repo2", "table2", None], "as tbl2 where ...")]
```

Comment:
That's odd. This causes the UT's to fail for me. Instead, when running the UT's, `self.query_rewriter.extract_table_token(token) == [(["repo1", "table1", None], "as tbl1"), (["repo2", "table2", None], "as tbl2")]`